docs/operators/goflow_input.md

```diff
@@ -0,0 +1,57 @@
+## `goflow_input` operator
+
+The `goflow_input` operator recieves Netflow v9 / IPFIX, Netflow v5, and Sflow messages from network devices. `goflow_input` implements [Goflow](https://github.com/cloudflare/goflow).
+
+The `timereceived` field is promoted as the entries Timestamp.
+
+### Configuration Fields
+
+| Field        | Default             | Description                                                                                   |
+| ---          | ---                 | ---                                                                                           |
+| `id`         | `goflow_input`      | A unique identifier for the operator                                                          |
+| `output`     | Next in pipeline    | The connected operator(s) that will receive all outbound entries                              |
+| `mode`       | required            | The Goflow mode [`netflow_ipfix`, `netflow_v5`, `sflow`]                                         |
```

Comment:
Is anyone of these protocols significantly more common that the others? If we believe so, it might be reasonable to set it as default.

Reply:
Good idea. Switched to netflow_ipfix. It is the latest protocol and supports v9 as well, half of the use cases.